### PR TITLE
Added --stdin option and fixed workers type issue

### PR DESCRIPTION
**Pull request by zAbuQasem**
Add an option to take the urls from stdin which is useful for using it with other tools, for example:
```bash
echo <DOMAIN> | waybackurls | unew | python3 sqlidetector.py --stdin -w 150 -o output.txt
```
Also added a simple if statement to filterout urls with no  ` ? ` (used it as an identifier for parameters)

### CODE REVIEW

File: sqlidetector.py [hunk 1]
```diff
@@ -1,12 +1,21 @@
 #!/usr/bin/python3
+from sys import stdin, argv
 from core.cli import cli_opts, print_logo
 from core.app import Scanner
+from rich.console import Console
 
+console = Console(style="bold red")
```
Comment (MindPatch) [inline sqlidetector.py]: do we really need this style option ?

Comment (zAbuQasem) [reply sqlidetector.py]: Nope, shall I remove it or what?

Comment (zAbuQasem) [reply sqlidetector.py]: @knassar702

**Comment by zAbuQasem** [comment]
The conflict doesn't affect the code

File: sqlidetector.py (hunk 1 above)
Comment (eslam3kl) [reply sqlidetector.py]: We will merge them but kindly remove the style option

Comment (zAbuQasem) [reply sqlidetector.py]: Done @eslam3kl

**Pull request merged**